test/unit/lib/mapIAPTResults.js

```diff
@@ -1,5 +1,5 @@
 const chai = require('chai');
-
+const constants = require('../../../app/lib/constants');
```

Comment:
This could be changed to used object destructuring e.g. `const { metrics } = require('../../../app/lib/constants');`